src/ui-fragment/src/components/ui-slider.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Slider } from 'carbon-components-react';
+import { CssClasses } from '../types';
+
+export interface SliderState {
+	type: string;
+	cssClasses?: CssClasses[];
+	labelText: string;
+	min: number;
+	max: number;
+	disabled?: boolean;
+	step?: number;
+	hideTextInput?: boolean;
+	light?: boolean;
+	value: number;
+	stepMultiplier?: number;
+	minLabel?: string;
+	maxLabel?: string;
+	codeContext?: {
+		name: string;
+	};
+}
+
+export const UISlider = ({ state }: {
+	state: SliderState;
+	setState: (state: any) => void;
+	setGlobalState: (state: any) => void;
+}) => {
+	if (state.type !== 'slider') {
+		// eslint-disable-next-line react/jsx-no-useless-fragment
+		return <></>;
+	}
+
+	return <Slider
+		id='slider'
+		labelText={state.labelText}
+		min={state.min}
+		max={state.max}
+		value={state.value}
+		disabled={state.disabled}
+		step={state.step}
+		hideTextInput={state.hideTextInput}
```

Comment:
see how state.hideTextInput makes it look like you're providing a reference to a function? `hideTextInput` _is_ a boolean, isn't it?